dietpi/dietpi-software

```diff
@@ -5197,7 +5197,7 @@ _EOF_
 
 			Banner_Installing
 
-			INSTALL_URL_ADDRESS='https://dietpi.com/downloads/binaries/all/netdata_1.9.0_'
+			INSTALL_URL_ADDRESS='https://dietpi.com/downloads/binaries/all/netdata_1.11.1-1_'
```

Comment:
@Fourdee 
What do you think about not including version string in the link? That way we can update the binary without the need to touch code, and, as well for older DietPi versions. Also then we don't need to keep all the older binaries on the server just to allow install from older DietPi versions.

Reply:
@MichaIng 

> What do you think about not including version string in the link? That way we can update the binary without the need to touch code, and, as well for older DietPi versions.

I'd prefer to keep the version codes, as if netdata 1.12 requires additional pre-reqs, or, different permissions, it would break DietPi v6.19 or lower installations.

Reply:
@Fourdee 
Jep just had the same idea while writing... Good point